### DISCUSSION TITLE
Custom toolkit - module not found

### DISCUSSION THREAD

**Discussion by neph1** [TransformerOptimus/SuperAGI#775] [Q&A] [answered]
I've made a custom toolkit that loads in the GUI.
It works in the local tool repo.
I can see in the logs that it runs through requirements.txt
So, everything looks OK.

But when I start an actual run with it, I get

`INFO/ForkPoolWorker-8] Task execute_agent[b6c09787-6fab-410a-8f0e-ff28bcf5ba96] retry: Retry in 0s: ModuleNotFoundError("No module named '*'")`

My hunch is that it's an issue with docker, somehow. I've seen some posts about adding PYTHONPATH to the dockerfile. But that shouldn't really be necessary for a stand-alone module?

Anyone has more info on this?

**Comment by neph1** [comment]
@luciferlinx101 It seems you're the person with most insight in tool development. Do you have any idea why this is? I've run --build again just to be on the safe side. Here is the repo: [https://github.com/neph1/FinnhubSuperAgi](https://github.com/neph1/FinnhubSuperAgi) .

**Reply by luciferlinx101** [reply]
@neph1 Hey will check this out at my end and would let you know.

**Comment by luciferlinx101** [comment, marked as answer]
Hey @neph1 There was some bug at our end. I have fixed it soon it will be out.
I tested your tool worked in my latest fix.

**Reply by neph1** [reply]
Thanks for testing it out!
